IdokladSdk.IntegrationTests/Tests/Clients/PriceListItem/PriceListItemTests.Async.cs

```diff
@@ -130,5 +130,20 @@ public async Task BatchDeleteAsync_SuccessfullyDeleted()
             // Assert
             Assert.True(data.First());
         }
+
+        [Test]
+        public async Task GetListWithSortAsync_ReturnsList()
+        {
+            // Act
+            var data = (await PriceListItemClient
+                .List()
+                .Sort(x => x.Name.Desc())
+                .GetAsync())
+                .AssertResult();
+
+            // Assert
+            Assert.NotNull(data);
+            Assert.Greater(data.Items.Count(), 0);
```

Comment:
Ověřuješ jen přítomnost dat. Nevíme, jestli jsou správně seřazená.
Zkusil bych něco takového: 
Assert.That(collection, Is.Ordered.By("PropertyName"));

Kdyby to nefungovalo, možná zafunguje jiná konstrukce odsud: https://stackoverflow.com/questions/1940214/how-to-check-if-a-list-is-ordered

Reply:
ok, pridám